ftw/oidcauth/browser/oidc_tools.py

```diff
@@ -16,6 +15,7 @@
 from jwt.exceptions import DecodeError
 from jwt.exceptions import InvalidTokenError
 from plone import api
+from plone.dexterity.utils import safe_unicode
```

Comment:
Is `plone.dexterity` a dependency of this package? If not, there is another implementation in `Products.CMFPlone.utils.safe_unicode`.

Reply:
It's not a direct dependency but required by:

**Plone 4.3**
- plone.app.layout
- plone.app.dexterity
- plone.app.uuid

**Plone 5.1**
- plone.app.layout
- plone.app.multilingual
- plone.app.intid
- plone.app.contenttypes
- plone.app.dexterity
- plone.app.versioningbehavior
- plone.app.uuid
- plone.app.lockingbehavior
- plone.app.event